src/main/java/org/gridsuite/actions/server/ContingencyListService.java

```diff
@@ -416,7 +240,7 @@ public ScriptContingencyList newScriptFromFormContingencyList(UUID id, UUID newI
         Optional<FormContingencyListEntity> formContingencyList = self.doGetFormContingencyListWithPreFetchedCountries(id);
         return formContingencyList.map(entity -> {
             String script = generateGroovyScriptFromForm(fromFormContingencyListEntity(entity));
-            ScriptContingencyListEntity scriptEntity = new ScriptContingencyListEntity(new ScriptContingencyList(null, script));
+            ScriptContingencyListEntity scriptEntity = new ScriptContingencyListEntity(new ScriptContingencyList(null, null, script));
```

Comment:
We have a strange case here where we could have a ScriptContingencyList with null Id (or generated then ?) and the ScriptContingencyListEntity with another id (cf `newId == null ? UUID.randomUUID() : newId`) this seems error prone

Reply:
Hum yes. If the newId is null, we should not create the contingency list. Or there's a case for such behavior ?

Reply:
But not the topid of this PR

Reply:
agreed